### ISSUE TITLE
TypeError: Cannot read property 'pop' of undefined

### ISSUE THREAD

**Issue by 0x6431346e** [aragon/aragen#61]
## 🐛 Bug Report

```sh
realisations-MacBook-Pro:etude realisation$ aragon devchain --debug 
Starting a local chain from snapshot [started]
Local chain started at port 8545 [title changed]

/Users/realisation/.nvm/versions/node/v11.12.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:360
  var lastNode = stack.pop()
                       ^
TypeError: Cannot read property 'pop' of undefined
    at CheckpointTrie.Trie._updateNode (/Users/realisation/.nvm/versions/node/v11.12.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:360:24)
    at /Users/realisation/.nvm/versions/node/v11.12.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:107:16
```

**Comment by realisation** [comment]
So it has something to do with the snapshot, I think.

I've been making devchain work by running `npx @aragon/aragen@4.1.2 devchain`, but as I was developing and tried to install a token-manager into my dao, it said it couldn't find it in the registry. So I thought that it had something to do with the snapshot I was using. 

I changed the snapshot to the one that came with `4.2.0` while still running with `4.1.2`, and I went straight back to getting the pop of undefined errors.
